lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html

```diff
@@ -93,7 +93,9 @@
         class="adf-textitem-clickable"
         [ngClass]="{ 'adf-property-read-only': !isEditable }"
         [attr.data-automation-id]="'card-textitem-toggle-' + property.key"
-        (click)="clicked()">
+         tabindex="0"
```

Comment:
if you compare these new attributes with these above then you can notice that indentation is a bit incorrect, please correct